### PR TITLE
feat: added close callout btn

### PR DESCRIPTION
**Pull request by brokewhale**
added callout close btn

https://github.com/user-attachments/assets/8cd8a0e8-b101-4e1e-a5c4-ce627c24571e

### CODE REVIEW

**Review by krofax — APPROVED** [review]

File: components/WipCallout.tsx [hunk 1]
```diff
@@ -67,8 +81,12 @@ export function AltCallout({ context }: Props): ReactElement {
           context
         ) : (
           <div className="nx-text-left">
-            The Alt-DA Mode feature is currently in <strong>Beta</strong>  within the MIT-licensed OP Stack. Beta features are built and reviewed by the Optimism Collective’s core contributors, and provide developers with early access to highly requested configurations.
-These features may experience stability issues, and we encourage feedback from our early users.
+            The Alt-DA Mode feature is currently in <strong>Beta</strong> within
```
Comment (bradleycamacho) [inline components/WipCallout.tsx:84]: Can we get these into a single line again? Or did they get split up for a reason?

**Review by bradleycamacho — CHANGES_REQUESTED** [review]
Looks great! As long as the button is actually clickable on mobile and you check out my small comment, this looks great to me! Thanks!!

File: components/WipCallout.tsx (hunk 1 above)
Comment (brokewhale) [reply components/WipCallout.tsx:84]: ohh just seeing this, i think this might be the result of the linting

**Comment by bradleycamacho** [comment]
> Looks great! As long as the button is actually clickable on mobile and you check out my small comment, this looks great to me! Thanks!!

Can confirm the button works on mobile btw

File: components/WipCallout.tsx (hunk 1 above)
Comment (brokewhale) [reply components/WipCallout.tsx:84]: but it should still render the same way regardless

Comment (brokewhale) [reply components/WipCallout.tsx:84]: mobile confirmed

https://github.com/user-attachments/assets/cefdc8a1-08e9-4e38-8751-84f8942398cb

**Review by brokewhale — COMMENTED** [review]
✅

**Comment by krofax** [comment]
@bradleycamacho please approval and merge.

**Pull request merged**